### PR TITLE
Chunksize

### PR DESCRIPTION
**Pull request by phunkyfish**
v1.8.0
- Added: Support Chunk (a.k.a Block) size
- Update: Remove pthreads library as now w32pthreads is used instead

Implemented in IPTV Simple: https://github.com/kodi-pvr/pvr.iptvsimple/pull/348

### CODE REVIEW

File: src/StreamManager.cpp [hunk 1]
```diff
@@ -301,6 +308,12 @@ bool CInputStreamLibavformat::IsRealTimeStream()
   return m_stream->IsRealTimeStream();
 }
 
+int CInputStreamLibavformat::GetBlockSize() 
+{
+  Log(LOGLEVEL_NOTICE, "Block Size(): %d", m_streamReadChunkSizeKb);
```
Comment (phunkyfish) [inline src/StreamManager.cpp:313]: @AlwinEsch strangely this never get's called and I don't know why. It is logged as a stream property when the inputstream is opened.

Comment (AlwinEsch) [reply src/StreamManager.cpp:313]: BlockSize only be used if demuxing in Kodi itself is used and addon not provide if `INPUTSTREAM_SUPPORTS_IDEMUX` set by addon.

Comment (AlwinEsch) [reply src/StreamManager.cpp:313]: You can leave away from this addon :smile:

Comment (phunkyfish) [reply src/StreamManager.cpp:313]: Cool

**Pull request closed without merging**